lessons/31-js-data-properties/index.js

```diff
@@ -0,0 +1,9 @@
+
+let favouriteMovies=prompt("Enter your 3 favourite movies separated by commas:");
+console.log(favouriteMovies.split(','));
+
+let stringToReverse= prompt("Enter a sentence u want to reverse");
+function reverseString(){
+  return stringToReverse.split(' ,').reverse().join(' ');
+}
+alert(`the reversed string is: ${reverseString()}` )
```

Comment:
Looks good! The only suggestion I would make is to take in the string to reverse as a parameter of `reverseString()` so that it is more reusable.

Also would likely want to do `.split('')` (with no comma) so that it goes character-by-character